numpy/core/include/numpy/experimental_dtype_api.h

```diff
@@ -321,6 +327,28 @@ typedef int (PyArrayMethod_StridedLoop)(PyArrayMethod_Context *context,
         NpyAuxData *transferdata);
 
 
+/**
+ * Query an ArrayMethod for the initial value for use in reduction.
+ *
+ * @param context The arraymethod context, mainly to access the descriptors.
+ * @param reduction_is_empty Whether the reduction is empty, when it is the
+ *     default value is required, otherwise an identity value to start the
+ *     the reduction.  These might differ, examples:
+ *     - `0.0` as default for `sum([])`.  But `-0.0` would be the correct
+ *       identity as it preserves the sign for `sum([-0.0])`.
+ *     - We use no identity for object, but `0` and `1` for sum and prod.
+ *     - `-inf` or `INT_MIN` for `max` is an identity, but at least `INT_MIN`
+ *       not a good *default* when there are no items.
+ * @param initial Pointer to initial data to be filled (if possible)
+ *
+ * @returns -1, 0, or 1 indicating error, no initial value, and initial being
+ *     successfully filled.  Errors must not be given where 0 is correct, NumPy
+ *     may call this even when not strictly necessary.
+ */
+ #define NPY_METH_get_reduction_initial 4
```

Comment:
It looks like you updated the values of the other `NPY_METH` constants in array_method.h, but not here.

Maybe it would help to catch stuff like this if the scaled float dtype used this header.